astropy/utils/iers/tests/test_iers.py

```diff
@@ -107,18 +107,18 @@ def test_simple(self):
         assert status[0] == iers.FROM_IERS_B
         assert np.all(status[1:] == iers.FROM_IERS_A)
         assert_quantity_allclose(ut1_utc,
-                                 [-0.4916557, -0.4925323, -0.4934373] * u.s,
-                                 atol=1.*u.ns)
+                                 [-0.491659, -0.492541, -0.4934373] * u.s,
+                                 atol=1e-6 * u.s)
```

Comment:
Note that the assertion that the last **two** values are from IERS_A is actually wrong if you look at the `iers_a_excerpt` file.  This has to do with the `i` index being from the `right` side `searchsorted` and there being a bit of an off-by-one error there.  I think this is a minor bug that can be dealt with later, but I was surprised originally to see two of the values changing.